edgecast/rtld/lookups/lookups_client.go

```diff
@@ -8,19 +8,20 @@ package lookups
 import (
 	"fmt"
 
-	"github.com/EdgeCast/ec-sdk-go/edgecast/client"
+	"github.com/EdgeCast/ec-sdk-go/edgecast/internal/ecclient"
 )
 
 // New creates a new lookups API client.
-func New(c client.Client) ClientService {
-	return &Client{Client: c}
+func New(c ecclient.APIClient, cc ecclient.ClientConfig) ClientService {
+	return &Client{c, cc}
 }
 
 /*
 Client for lookups API
 */
 type Client struct {
-	client.Client
+	client ecclient.APIClient
+	config ecclient.ClientConfig
```

Comment:
Since what's really needed is just the Base API URL, perhaps we don't need to keep the entire `ClientConfig` object here, and just keep `BaseAPIURL string` ?

Reply:
Updated.